.github/workflows/README.md

```diff
@@ -0,0 +1,67 @@
+# Toolbelt Workflow
+
+Goal: A mechanism that implements as many OpenSSF best practices using OpenSSF tools as possible in a way that is simple to use for project maintainers and easy for end users to check projects using this mechanism are following best practices and creating important security metadata (e.g. SBOMs, in-toto SLSA, etc.)
```

Comment:
As I understood it, the goal goes beyond "simple use" and is really about requiring minimal changes to existing CI/CD-based builds. If so, the text should probably be a bit more explicit about this. Also, on the end user side, how will the workflow actually help users check projects adherence to best practices? Is seeing that they use the workflow enough of a signal, or is there more they'd have to do to gain assurances?

Reply:
+1 on defining this a bit better. Also better defining if this is meant to be used as a reference architecture, playbook or aimed for production use have fairly different implications on the design decisions in the toolbelt project.

Reply:
@lumjjb I think in this case a lot of it "depends" -- Ideally we would love to have this be something that is used in production by anyone who has a use case supported by the toolbelt. The issue is finding folks who would use this.

We've gotten folks who have said they're interested but actually getting them to use it is a different story.

Reply:
Notes from meeting:
- We want this to be in CI/CD since that is the most critical part of the SDLC for stopping malicious stuff from getting in.
- This workflow should ideally focus in Github actions but could expand beyond that.
- Ideally we want this used as the de facto build workflow.
- Here is an example: https://github.com/mlieberman85/skootrs-updates-8/blob/main/.github/workflows/toolbelt.yml